src/ProcessMaker/Nayra/Bpmn/TransitionTrait.php

```diff
@@ -97,6 +98,19 @@ protected function conditionIsFalse()
      */
     protected function doTransit(CollectionInterface $consumeTokens, ExecutionInstanceInterface $executionInstance)
     {
+        if ($this instanceof ConditionedExclusiveTransition
+            && !empty($this->getOwner()->getOwnerDocument())) {
```

Comment:
I think this condition is not required anymore:

`!empty($this->getOwner()->getOwnerDocument())`

Reply:
Yes, the condition it isn't necessary now. It was removed